src/test/java/com/training/api/tests/shorex/ShorexTests.java

```diff
@@ -64,4 +68,28 @@ public void testValidateProductTypeName() {
                     .isEqualTo("Shore Excursion");
         }
     }
+
+    @Test
+    public void testValidateOfferingDate() throws ParseException {
+
+        String pattern = "yyyyMMdd";
+        String stringDate = "";
+        Date searchDate = new SimpleDateFormat(pattern)
```

Comment:
directly write in line 93

Reply:
@PranavHanwante wont that take more resource? it has to retrieve the string from map and perform manipulations to get date each time to get a date that doesn't change